parrot/src/pfs_table.cc

```diff
@@ -2056,7 +2107,7 @@ int pfs_table::locate( const char *n, char *buf, int length )
 		if(loc) delete(loc);
 		loc = 0;
 
-		if(resolve_name(0, n, &pname)) {
+		if(resolve_name(0, n, &pname, X_OK)) { //TODO check this
```

Comment:
I think generally just use F_OK unless there's a clear need for X_OK. chdir/listdir are the only ones that come to mind for me.